// @jablonowski/tokens/dist/ng-bank/ts/tokens.ts
/**
 * Do not edit directly, this file was auto-generated.
 */

export const BorderWidthNone = "0";
export const BorderWidthThin = "1px";
export const BorderWidthMedium = "2px";
export const BorderWidthThick = "4px";
export const BorderRadiusNone = "0";
export const BorderRadiusSm = "0.25rem";
export const BorderRadiusMd = "0.5rem";
export const BorderRadiusLg = "1rem";
export const BorderRadiusXl = "1.5rem";
export const BorderRadiusFull = "9999px";
export const BreakpointXs = "0";
export const BreakpointSm = "576px";
export const BreakpointMd = "768px";
export const BreakpointLg = "992px";
export const BreakpointXl = "1200px";
export const Breakpoint2xl = "1400px";
export const ColorPrimary50 = "#fff3e0";
export const ColorPrimary100 = "#ffe0b2";
export const ColorPrimary200 = "#ffcc80";
export const ColorPrimary300 = "#ffb74d";
export const ColorPrimary400 = "#ffa726";
export const ColorPrimary500 = "#FF6200";
export const ColorPrimary600 = "#e55800";
export const ColorPrimary700 = "#bf4900";
export const ColorPrimary800 = "#993a00";
export const ColorPrimary900 = "#732c00";
export const ColorPrimaryContrast50 = "rgba(0, 0, 0, 0.87)";
export const ColorPrimaryContrast100 = "rgba(0, 0, 0, 0.87)";
export const ColorPrimaryContrast200 = "rgba(0, 0, 0, 0.87)";
export const ColorPrimaryContrast300 = "rgba(0, 0, 0, 0.87)";
export const ColorPrimaryContrast400 = "rgba(0, 0, 0, 0.87)";
export const ColorPrimaryContrast500 = "#ffffff";
export const ColorPrimaryContrast600 = "#ffffff";
export const ColorPrimaryContrast700 = "#ffffff";
export const ColorPrimaryContrast800 = "#ffffff";
export const ColorPrimaryContrast900 = "#ffffff";
export const ColorSecondary50 = "#eae7f6";
export const ColorSecondary100 = "#c5bfe8";
export const ColorSecondary200 = "#9f96d9";
export const ColorSecondary300 = "#7a6ecb";
export const ColorSecondary400 = "#6050c0";
export const ColorSecondary500 = "#525199";
export const ColorSecondary600 = "#49488a";
export const ColorSecondary700 = "#3e3d75";
export const ColorSecondary800 = "#333261";
export const ColorSecondary900 = "#23213f";
export const ColorSecondaryContrast50 = "rgba(0, 0, 0, 0.87)";
export const ColorSecondaryContrast100 = "rgba(0, 0, 0, 0.87)";
export const ColorSecondaryContrast200 = "rgba(0, 0, 0, 0.87)";
export const ColorSecondaryContrast300 = "rgba(0, 0, 0, 0.87)";
export const ColorSecondaryContrast400 = "#ffffff";
export const ColorSecondaryContrast500 = "#ffffff";
export const ColorSecondaryContrast600 = "#ffffff";
export const ColorSecondaryContrast700 = "#ffffff";
export const ColorSecondaryContrast800 = "#ffffff";
export const ColorSecondaryContrast900 = "#ffffff";
export const ColorNeutral0 = "#ffffff";
export const ColorNeutral50 = "#f5f5f5";
export const ColorNeutral100 = "#eeeeee";
export const ColorNeutral200 = "#e0e0e0";
export const ColorNeutral300 = "#bdbdbd";
export const ColorNeutral400 = "#9e9e9e";
export const ColorNeutral500 = "#757575";
export const ColorNeutral600 = "#616161";
export const ColorNeutral700 = "#424242";
export const ColorNeutral800 = "#303030";
export const ColorNeutral900 = "#1a1a1a";
export const ColorSuccessLight = "#e8f5e9";
export const ColorSuccessMain = "#4caf50";
export const ColorSuccessDark = "#1b5e20";
export const ColorWarningLight = "#fff8e1";
export const ColorWarningMain = "#ff9800";
export const ColorWarningDark = "#e65100";
export const ColorErrorLight = "#ffebee";
export const ColorErrorMain = "#d32f2f";
export const ColorErrorDark = "#b71c1c";
export const ColorInfoLight = "#e3f2fd";
export const ColorInfoMain = "#1976d2";
export const ColorInfoDark = "#0d47a1";
export const ColorBackgroundPrimary = "#ffffff";
export const ColorBackgroundSecondary = "#f5f5f5";
export const ColorBackgroundDark = "#2D2926";
export const ColorTextPrimary = "#1a1a1a";
export const ColorTextSecondary = "#616161";
export const ColorTextDisabled = "#9e9e9e";
export const ColorTextInverse = "#ffffff";
export const ElevationNone = "none";
export const ElevationXs = "0 1px 2px 0 rgba(0, 0, 0, 0.05)";
export const ElevationSm =
  "0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px -1px rgba(0, 0, 0, 0.06)";
export const ElevationMd =
  "0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -2px rgba(0, 0, 0, 0.06)";
export const ElevationLg =
  "0 10px 15px -3px rgba(0, 0, 0, 0.08), 0 4px 6px -4px rgba(0, 0, 0, 0.06)";
export const ElevationXl =
  "0 20px 25px -5px rgba(0, 0, 0, 0.08), 0 8px 10px -6px rgba(0, 0, 0, 0.06)";
export const Elevation2xl = "0 25px 50px -12px rgba(0, 0, 0, 0.2)";
export const SpacingNone = "0";
export const Spacing2xs = "0.125rem";
export const SpacingXs = "0.25rem";
export const SpacingSm = "0.5rem";
export const SpacingMd = "1rem";
export const SpacingLg = "1.5rem";
export const SpacingXl = "2rem";
export const Spacing2xl = "3rem";
export const Spacing3xl = "4rem";
export const Spacing4xl = "6rem";
export const FontFamilyBase =
  "'ING Me', 'Segoe UI', Roboto, Helvetica, Arial, sans-serif";
export const FontFamilyHeading =
  "'ING Me', 'Segoe UI', Roboto, Helvetica, Arial, sans-serif";
export const FontFamilyMonospace =
  "'SF Mono', 'Fira Code', 'Courier New', monospace";
export const FontSizeXs = "0.75rem";
export const FontSizeSm = "0.875rem";
export const FontSizeBase = "1rem";
export const FontSizeMd = "1.125rem";
export const FontSizeLg = "1.25rem";
export const FontSizeXl = "1.5rem";
export const FontSize2xl = "1.875rem";
export const FontSize3xl = "2.25rem";
export const FontSize4xl = "3rem";
export const FontWeightLight = "300";
export const FontWeightRegular = "400";
export const FontWeightMedium = "500";
export const FontWeightSemibold = "600";
export const FontWeightBold = "700";
export const FontLineHeightTight = "1.25";
export const FontLineHeightBase = "1.5";
export const FontLineHeightLoose = "1.75";
export const FontLetterSpacingTight = "-0.025em";
export const FontLetterSpacingNormal = "0em";
export const FontLetterSpacingWide = "0.025em";
export const FontLetterSpacingWider = "0.05em";
